addons/web_editor/models/ir_ui_view.py

```diff
@@ -81,6 +81,7 @@ def save_oe_structure(self, el):
             'arch': self._pretty_arch(arch),
             'key': '%s_%s' % (self.key, el.get('id')),
             'type': 'qweb',
+            'mode': 'extension'
```

Comment:
Oh... indeed but why does it work for some oe_structure then? :scream:

Reply:
@hsi-odoo In fact I see that in the case where it works, the mode is indeed set to "extension" automatically... do you know why?

Reply:
@hsi-odoo We found the issue, in fact it is because the create is done using `self` two lines below. Could you also change that into `self.env['ir.ui.view'].create` to make it more explicit? Thanks :)